com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.LightLoop.cs

```diff
@@ -40,6 +40,15 @@ static void BindGlobalLightingBuffers(in LightingBuffers buffers, CommandBuffer
             cmd.SetGlobalTexture(HDShaderIDs._ScreenSpaceShadowsTexture, buffers.screenspaceShadowBuffer);
         }
 
+        static void BindDefaultTexturesLightingBuffers(RenderGraphDefaultResources defaultResources, CommandBuffer cmd)
```

Comment:
I'm not sure I'm a fan of binding everything globally when the plan is to remove all globals like this.
Can't we just return the proper default texture from the right passes as we do some other places?

Reply:
This just mirrors the others that are bound globally (see a bit above in the file), the problem is that the set global and all the passes producing the outputs are skipped during debug. 

The passes already return the right default it is just that they are never run :D

Reply:
Fair enough. I'll take a deeper look when I actually do the thing.